### ISSUE TITLE
shall we allow survey creator to fill survey on their own?

### ISSUE THREAD

**Issue by lmmrssa** [open-learning-exchange/planet#2250]
Right now survey creator is not allowed to submit survey unless he does it by record survey or someone else send it to him. Shall we allow them to self submit?
May be adding survey automatically to list or having button to submit survey if he is owner of it.

**Comment by paulbert** [comment]
Yes we should.  Need to modify DialogsListService to allow this for only this instance.
